doc/filters/join.rst

```diff
@@ -17,6 +17,18 @@ define it with the optional first parameter:
     {{ [1, 2, 3]|join('|') }}
     {# outputs 1|2|3 #}
 
+Combining the range operator with the join filter allows to output that same
+range of elements without having to define all of them:
+
+.. code-block:: jinja
+
+    {{ (1..3)|join('|') }}
+    {# outputs 1|2|3 #}
+    
+    {% set variable = 6 %}
+    {{ (1..variable)|join('|') }}
+    {# outputs 1|2|3|4|5|6 #}
```

Comment:
I don't understand what this is about. The above example `{{ [1, 2, 3]|join }}` could be anything else like `{{ [22, 42, 54]|join }}` and then the added example would not make sense anymore, right? If the fact the 1, 2, 3 is a sequence is disturbing, let's change it to some more random numbers. The docs try to document one filter/function at a time and only mention other ones when there is a relation, and that's not the case here.